erpnext/stock/doctype/pick_list/pick_list.json

```
@@ -71,8 +71,7 @@
    "description": "Qty of raw materials will be decided based on the qty of the Finished Goods Item",
    "fieldname": "for_qty",
    "fieldtype": "Float",
-   "label": "Qty of Finished Goods Item",
-   "read_only": 1
+   "label": "Qty of Finished Goods Item"
```

```suggestion
   "label": "Qty of Finished Goods Item",
   "read_only": 1
```
What is the rationale for making this editable?